### ISSUE TITLE
Add Retopology Operators break RF if run via Shift+A menu

### ISSUE THREAD

**Issue by vxlcoder** [CGCookie/retopoflow#1322]
Issue #1321 had a secondary issue that I'm capturing here.

The Add Retopology at Cursor / Active Operators through the Shift+A menu instead of the header menu does absolutely nothing.  The following error message would be silently reported to the console.

```
ERROR (wm.operator): source/blender/windowmanager/intern/wm_event_system.cc:1571 wm_operator_invoke: invalid operator call 'RETOPOFLOW_OT_newtarget_cursor'
```

The issue is due to the operator defining an `invoke` method but not an `execute` method.

**Comment by vxlcoder** [comment]
this issue appears to be fixed now!

Thanks to @JornBoven for helping debug this issue!

You can use this temp link to test it out (same link as in issue #1321): https://www.dropbox.com/scl/fi/tgbmx8b690orvcc31yp7a/RetopoFlow_v4.0.0-alpha-GitHub.zip?rlkey=380tjledg3h3o0c9g3xc1vloo&st=34u2ntfs&dl=0

**Comment by JornBoven** [comment]
I can confirm that it works on my side too. Thank you for your attention to this issue~ I appreciate it!

**Comment by jfranmatheu** [comment]
Hey @vxlcoder This issue seems fixed with da40941 but the issue itself is still in open state
